packages/cosmic-swingset/src/sim-chain.js

```diff
@@ -197,7 +202,7 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
     // Only actually simulate a block if we're not in bootstrap.
     let p;
     if (blockHeight && !delay) {
-      clearTimeout(nextBlockTimeout);
+      if (nextBlockTimeout !== undefined) clearTimeout(nextBlockTimeout);
```

Comment:
We probably should reset here

```suggestion
      if (nextBlockTimeout !== undefined) {
        clearTimeout(nextBlockTimeout);
        nextBlockTimeout = undefined;
      }
```